types/src/on_chain_config/aptos_features.rs

```diff
@@ -36,7 +36,8 @@ pub enum FeatureFlag {
     MODULE_EVENT = 26,
     EMIT_FEE_STATEMENT = 27,
     STORAGE_DELETION_REFUND = 28,
-    SIGNATURE_CHECKER_V2_SCRIPT_FIX = 29,
+    AGGREGATOR_SNAPSHOTS = 29,
+    SIGNATURE_CHECKER_V2_SCRIPT_FIX = 30,
```

Comment:
Where does this come from?

Reply:
actually, let,s make sure AGGREGATOR_SNAPSHOTS = 30, and SIGNATURE_CHECKER_V2_SCRIPT_FIX = 29. Just in case @vusirikala

Reply:
Done.